cli/internal/cmd/upgradeapply.go

```diff
@@ -124,7 +138,11 @@ func (u *upgradeApplyCmd) upgradeApply(cmd *cobra.Command, fileHandler file.Hand
 	conf.UpdateMAAURL(idFile.AttestationURL)
 
 	// If an image upgrade was just executed there won't be a diff. The function will return nil in that case.
-	if err := u.upgradeAttestConfigIfDiff(cmd, conf.GetAttestationConfig(), flags); err != nil {
+	stableClient, err := stableClientFactory(adminConfPath(flags.workspace))
```

Comment:
```suggestion
	stableClient, err := stableClientFactory(constants.AdminConfFilename)
```
The workspace paths are just for printing correct filenames to the user